data-structures-stacks/get-top.js

```diff
@@ -0,0 +1,9 @@
+import './lib/stack';
+
+export default function getTop(stack) {
+  if (stack === '' || null) {
+    return undefined;
```

Comment:
Given `stack` will be an instance of the `Stack` class, I don't think it'll ever be an empty string (`''`). `null` is always falsey, so it doesn't appear that line 5 will ever run.